### PR TITLE
readme: normalize and tweak style

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/mem/issues/24 and parts of https://github.com/Level/mem/issues/20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,53 @@
...
+Use this package to avoid having to explicitly install `memdown` when you want to use `memdown` with `levelup` for non-persistent `levelup` data storage.
 
 ```js
-var level = require('level-mem')
+const level = require('level-mem')
 
 // 1) Create our database, supply location and options.
 //    This will create or open the underlying LevelDB store.
-var db = level('w00t')
+const db = level()
```
Comment (ralphtheninja) [inline README.md]: @vweevers I'm assuming you meant this `location`

Comment (vweevers) [reply README.md]: Exactly. And the comment above it ("supply location")

Comment (ralphtheninja) [reply README.md]: Oh right.

File: README.md [hunk 2]
```diff
@@ -1,55 +1,53 @@
...
-**A convenience package bundling LevelUP & MemDOWN**
+[![level badge][level-badge]](https://github.com/level/awesome)
+[![npm](https://img.shields.io/npm/v/level-mem.svg)](https://www.npmjs.com/package/level-mem)
+[![Build Status](https://secure.travis-ci.org/Level/mem.png)](http://travis-ci.org/Level/mem)
+[![david](https://david-dm.org/Level/mem.svg)](https://david-dm.org/level/mem)
+[![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
+[![npm](https://img.shields.io/npm/dm/level-mem.svg)](https://www.npmjs.com/package/level-mem)
 
-[![NPM](https://nodei.co/npm/level-mem.png)](https://nodei.co/npm/level-mem/)
+This is a convenience package that bundles the current release of [`levelup`](https://github.com/level/levelup) and [`memdown`](https://github.com/level/memdown) and exposes `levelup` on its export.
```
Comment (vweevers) [inline README.md]: Maybe merge this sentence into the short description above, it's redundant except for the "and exposes levelup ..".

File: README.md [hunk 3]
```diff
@@ -1,55 +1,51 @@
-level-mem
-=========
+# level-mem
 
-<img alt="LevelDB Logo" height="100" src="http://leveldb.org/img/logo.svg">
+> A convenience package bundling [`levelup`](https://github.com/level/levelup) and [`memdown`](https://github.com/level/memdown) and exposes `levelup` on its export.
```
Comment (vweevers) [inline README.md]: Either "bundling ... and exposing" or "that bundles .. and exposes"

Comment (vweevers) [reply README.md]: The rest LGTM. Go ahead and merge after this.

**Pull request merged**